### PR TITLE
Issue #42 coala-analyzer.org --> coala.io at artwork/ads/joinus.svg

### PR DESCRIPTION
**Pull request by abhayKashyap03**

### CODE REVIEW

**Comment by Vamshi99** [comment]
@abhayKashyap03 Your change is good, you only need to do change your commit message. Go through https://api.coala.io/en/latest/Developers/Writing_Good_Commits.html#how-to-write-good-commit-messages.

Note: to change the commit message, just type `git commit --amend` in your git terminal and it will open an editor to change your commit message. After the change, do force push using `git push -f origin your_branch`. Or go through https://api.coala.io/en/latest/Developers/Git_Basics.html#follow-up.

File: artwork/ads/joinus.svg [hunk 1]
```diff
@@ -624,4 +624,4 @@
          y="0"
          sodipodi:role="line"
          id="tspan456">Javascript | Java | Python | HTML | C++ | CSS</tspan></text>
-</g></svg>
```
Comment (Vamshi99) [inline artwork/ads/joinus.svg:627]: @abhayKashyap03 Unnecessary change. Remove this

**Comment by Vamshi99** [comment]
You have 3 commits in this PR, squash them all into a single commit, for help checkout https://api.coala.io/en/latest/Developers/Git_Basics.html#squashing-your-commits

File: artwork/ads/joinus.svg (hunk 1 above)
Comment (abhayKashyap03) [reply artwork/ads/joinus.svg:627]: This was not a change I made, it must have happened accidentally.

**Comment by abhayKashyap03** [comment]
Can I close this and create a new PR?
There are some changes that have happened accidentally and I just want to remove them.

**Comment by ayan-b** [comment]
Never create a new PR, please fix everything here only.

**Comment by jayvdb** [comment]
Another GCI participant has done this task.
